src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -112,6 +112,10 @@ public function getProxyCode(Definition $definition)
             );
         }
 
+        if (version_compare(self::getProxyManagerVersion(), '2.5', '<')) {
+            $code = str_replace('\Closure::bind(function ', '\Closure::bind(static function ', $code);
```

Comment:
missing space at the beginning:
`str_replace(' \Closure::bind(function ', ' \Closure::bind(static function ', $code);`